[BUG] Flaky test org.opensearch.action.admin.ClientTimeoutIT.testSegmentReplicationStatsWithTimeout

**Describe the bug**
- Getting following error:

```
  2> java.lang.AssertionError: timed out waiting for green state
        at __randomizedtesting.SeedInfo.seed([CA9A15EE7B62C96A:56C5C8CB0C0195B3]:0)
        at org.junit.Assert.fail(Assert.java:89)
        at org.opensearch.test.OpenSearchIntegTestCase.ensureColor(OpenSearchIntegTestCase.java:1001)
        at org.opensearch.test.OpenSearchIntegTestCase.ensureGreen(OpenSearchIntegTestCase.java:932)
        at org.opensearch.test.OpenSearchIntegTestCase.ensureGreen(OpenSearchIntegTestCase.java:921)
        at org.opensearch.action.admin.ClientTimeoutIT.testSegmentReplicationStatsWithTimeout(ClientTimeoutIT.java:177)
```

**To Reproduce**
```
./gradlew ':server:internalClusterTest' --tests "org.opensearch.action.admin.ClientTimeoutIT.testSegmentReplicationStatsWithTimeout" -Dtests.seed=CA9A15EE7B62C96A -Dtests.security.manager=true -Dtests.jvm.argline="-XX:TieredStopAtLevel=1 -XX:ReservedCodeCacheSize=64m" -Dtests.locale=is-IS -Dtests.timezone=Africa/Monrovia -Druntime.java=19
```

--- a/rest-api-spec/src/main/resources/rest-api-spec/api/cat.segment_replication.json
+++ b/rest-api-spec/src/main/resources/rest-api-spec/api/cat.segment_replication.json
@@ -4,7 +4,7 @@
       "url":"https://github.com/opensearch-project/documentation-website/issues/2627",
       "description":"Returns information about both on-going and latest completed Segment Replication events"
     },
-    "stability":"stable",
+    "stability":"experimental",
     "url":{
       "paths":[
         {

--- a/server/src/internalClusterTest/java/org/opensearch/action/admin/ClientTimeoutIT.java
+++ b/server/src/internalClusterTest/java/org/opensearch/action/admin/ClientTimeoutIT.java
@@ -152,6 +152,7 @@ public class ClientTimeoutIT extends OpenSearchIntegTestCase {
         assertThat(recoveryResponse.getShardFailures()[0].reason(), containsString("ReceiveTimeoutTransportException"));
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/6255")
     public void testSegmentReplicationStatsWithTimeout() {
         internalCluster().startClusterManagerOnlyNode(
             Settings.builder().put(super.featureFlagSettings()).put(FeatureFlags.REPLICATION_TYPE, "true").build()

